src/main/kotlin/app/revanced/patcher/Patcher.kt

```diff
@@ -39,6 +42,7 @@ class Patcher(private val options: PatcherOptions) {
     private var resourceDecodingMode = ResourceDecodingMode.MANIFEST_ONLY
     private var mergeIntegrations = false
     val context: PatcherContext
+    private lateinit var methodMap : Map<String, List<Pair<ClassDef, Method>>>
```

Comment:
This field shouldn't be present here and instead somewhere in the fingerprint classes, because it is used there